src/NzbDrone.Core/Datastore/Migration/140_remove_folderchmod_config.cs

```diff
@@ -0,0 +1,15 @@
+using System.Data;
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(140)]
+    public class remove_folderchmod_config : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Execute.Sql("DELETE FROM Config WHERE Key = 'folderchmod'");
```

Comment:
Note sure if we should merge it with filechmod. Like `filechmod = (filechmod | folderchmod) & ~(execute flags)`.
I don't expect anyone to have them configured differently, so it's probably not needed.

Reply:
Also delete chown user&group

Reply:
I think they shouldn't be merged, it may cause unexpected issues.
Sorry forgot the chown :)